Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationSelection.cshtml.cs

```diff
@@ -13,11 +14,31 @@ public class OrganisationSelectionModel(
 {
     public IEnumerable<Organisation.WebApiClient.Organisation> Organisations { get; set; } = [];
 
+    public string? Error { get; set; }
+
     public async Task<IActionResult> OnGet()
     {
         var details = VerifySession();
 
-        Organisations = await organisationClient.ListOrganisationsAsync(details.UserUrn);
+        try
+        {
+            Organisations = await organisationClient.ListOrganisationsAsync(details.UserUrn);
```

Comment:
List organisation response should never be 404

Reply:
true. I have updated this,